hopsworks-ear/test/spec/projects_spec.rb

```diff
@@ -340,6 +340,24 @@
           memb = json_body.detect { |e| e[:user][:email] == new_member }
           expect(memb).to be_nil
         end
+        it "should remove data scientist from project when remove issued by another data owner" do
```

Comment:
@robzor92 Do we have a test that should fail for a data scientist trying to remove a data owner? If not we need to add one

Reply:
yes adding one for that too

Reply:
added and now running tests on jenkins